docker/Dockerfile.dev

```diff
@@ -1,18 +1,21 @@
-FROM pcic/geospatial-python
+FROM pcic/geospatial-python:3.3.0
 
 RUN mkdir /app
-ADD requirements.txt /app
+ADD poetry.lock /app
+ADD pyproject.toml /app
 WORKDIR /app
-RUN pip3 install -U pip pipenv --ignore-installed
-RUN pipenv install
 
-ADD . /app/
+RUN apk add libffi-dev
+RUN wget -O - https://install.python-poetry.org | python3 -
+ENV PATH=/root/.local/bin:$PATH
+RUN poetry install --with=dev
+RUN poetry install
```

Comment:
Is this still necessary? What was the outcome of `install` vs `install --with`? The documentation sure looks as if it shouldn't be necessary. Also, the CI workflow doesn't use it. :thinking:

Reply:
Nope. Wasn't necessary after all. Thanks for checking.